docs/Common-issues.md

```diff
@@ -0,0 +1,14 @@
+## Python Issues OSX
+
+Make sure you are using the native Python version in OSX.  If you use a MacPorts of HomeBrew version, you may run into problems.
+
+If you have issues with `execvp`, be sure to check your `$PYTHON` environment variable.  If it is not set to the native version, unset it and try again.
+
+Notes: https://gist.github.com/erichocean/5177582
```

Comment:
This is about node-gyp v0.91 yet the current node-gyp is v8.1.  Is there any evidence that this problem still exists?